src/Core/Math/Ellipsoid.js

```diff
@@ -44,10 +45,10 @@ Ellipsoid.prototype.setSize = function setSize(size) {
 };
 
 
-Ellipsoid.prototype.cartographicToCartesian = function cartographicToCartesian(coordCarto) {
+Ellipsoid.prototype.cartographicToCartesian = function cartographicToCartesian(coordCarto, normal) {
```

Comment:
> Or alternatively (and I'd prefer this approach): you could return { xyz: k.add(n), normal: n} and adapt the other user of this function.

I have updated the code with this approach, and rebase because some changes was not needed any more. Thanks for the suggestion, it's more elegant.